source/application/VESPERS/VESPERSAppController.cpp

```diff
@@ -306,22 +305,6 @@ void VESPERSAppController::setupScanConfigurations()
 	timedLineScanConfiguration_->scanAxisAt(0)->regionAt(0)->setRegionTime(1.0);
 }
 
-void VESPERSAppController::setupUserConfiguration()
-{
-	connect(vespersUserConfiguration_, SIGNAL(loadedFromDb()), this, SLOT(onUserConfigurationLoadedFromDb()));
-
-	if (!vespersUserConfiguration_->loadFromDb(AMDatabase::database("user"), 1)){
-
-		vespersUserConfiguration_->storeToDb(AMDatabase::database("user"));
-		// This is connected here because our standard way for these signal connections is to load from db first, which clearly won't happen on the first time.
-		connect(VESPERSBeamline::vespers()->vespersSingleElementVortexDetector(), SIGNAL(addedRegionOfInterest(AMRegionOfInterest*)), this, SLOT(onRegionOfInterestAdded(AMRegionOfInterest*)));
```

Comment:
A number of detectors are connected to here. This code has been removed, and I'm not sure I see where the same connections are being made elsewhere.